### DISCUSSION TITLE
Document version

### DISCUSSION THREAD

**Discussion by xujinming01** [opendilab/DI-engine#663] [Q&A]
I have been reading the documentation of DI-Engine recently, which has both Chinese and English versions. I personally feel that the Chinese version is more up-to-date. Am I right in my understanding?

**Comment by PaParaZz1** [comment]
There are some differences between our Chinese and English version docs. For example, English doc has more algorithm introductions while Chinese docs contains more tutorials. And we are trying to add missing parts as soon as possible (e.g. [Call for Contributors](https://github.com/opendilab/DI-engine/issues/621)). Do you have other suggestions or need to supplement some specific doc?

**Reply by xujinming01** [reply]
Ok, thanks. I would like to implement some new algorithms based on DI Engine, and may open a PR if possible, so I am interested in [System Design](https://di-engine-docs.readthedocs.io/en/latest/03_system/index.html) and [Developer Specification](https://di-engine-docs.readthedocs.io/en/latest/20_spec/index.html).
